app/Application.php

```diff
@@ -74,57 +84,18 @@ public function run()
         }
     }
 
-    protected function calculate($argument1, $command, $argument2)
-    {
-        switch ($command) {
-            case '+':
-                return $argument1 + $argument2;
-            case '-':
-                return $argument1 - $argument2;
-            case '*':
-                return $argument1 * $argument2;
-            case '/':
-                return $argument1 / $argument2;
-            case '^':
-                return pow($argument1, $argument2);
-            case 'sr':
-                return pow($argument1, (1 / $argument2));
-            default:
-                return info($this->getText('errors', 'undefined_command', ['command' => $command]));
-        }
-    }
-
     protected function readOperand($message, $command, $isSecondOperand = false)
     {
-        do {
             $argument = readline($message);
```

Comment:
please fix formatting for further code as well

```suggestion
        $argument = readline($message);
```